src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Properties/AbstractProjectConfigurationProperties.cs

```diff
@@ -124,6 +124,27 @@ public string IntermediatePath
                 });
             }
         }
+        public bool RunCodeAnalysis
+        {
+            get
+            {
+                return _threadingService.ExecuteSynchronously(async () =>
+                {
+                    ConfiguredBrowseObject browseObjectProperties = await _projectProperties.GetConfiguredBrowseObjectPropertiesAsync();
+                    string value = await browseObjectProperties.RunCodeAnalysis.GetEvaluatedValueAtEndAsync();
+                    return string.Equals(value, bool.TrueString, StringComparisons.PropertyValues);
```

Comment:
GetValueAsync will return a (bool) here.

Reply:
Yeah, I wasn't 100% sure about this so I went with the assumption that the since all of the other properties in this file use `EvaluatedValueAtEnd` that it was more important that worrying too much about a conversion. Easy to change though.

Reply:
GetValue is probably the correct way to read these properties anyway - as the properties could be marked as "BeforeContext" (it's value as defined in the project file) and "AfterContext" (EvaluatedValueAtEnd) and we should be respecting this.

Reply:
Updated to use GetValueAsync. It indeed returns a `bool?`.